drivers/volume/portworx/portworx.go

```diff
@@ -3028,7 +2946,7 @@ func (p *portworx) createGroupLocalSnapFromPVCs(groupSnap *storkapi.GroupVolumeS
 	if err != nil {
 		return nil, err
 	}
-	resp, err := volDriver.SnapshotGroup("", nil, volNames)
+	resp, err := volDriver.SnapshotGroup("", nil, volNames, false)
```

Comment:
@disrani-px I added this `false` to let it compile after pulling dependencies. How can I understand that it doesn't break things?

Reply:
@talakad Is this the new flag that was added to clean up partially successful group snaps?

Reply:
Correct. https://github.com/libopenstorage/openstorage/blob/master/volume/volume.go#L115

Reply:
@talakad what default value should be used for portworx driver?

Reply:
This was added specifically for stork, so that stork need not follow up with delete commands for failed snaps in groupsnap request. For stork, this should be set to true.  @disrani-px ,  to comment otherwise. If clients don't mind some snaps failing , then it should be set to false.